src/gl/opengl/vcShader.cpp

```diff
@@ -243,7 +236,7 @@ bool vcShader_GetSamplerIndex(vcShaderSampler **ppSampler, vcShader *pShader, co
   if (pShader->numSamplerIndexes >= (int)udLengthOf(pShader->samplerIndexes))
     return false;
 
-  GLuint uID = glGetUniformLocation(pShader->programID, pSamplerName);
+  GLuint uID = glGetUniformLocation(pShader->programID, udTempStr("SPIRV_Cross_Combined%sTexture%sSampler", pSamplerName, pSamplerName));
 
   if (uID == GL_INVALID_INDEX)
     return true;
```

Comment:
You need to return `false` here, this was a hack to do the lookup in the `vcTexture_BindTexture` function, which you've removed.